### PR TITLE
#175636777 Support of multiple languages

### PR DESCRIPTION
**Pull request by fabriceumuhire**
### What does this PR do?
- Adding support of multiple languages
#### Description of Task to be completed?
- run `npm install i18n --save` to install i18n dependency
- Add three languages files to be supported (en.json, fr.json; ki.json)
- Configure the app (in index.js) file to support defined languages 
#### How should this be manually tested?
- run `git clone https://github.com/atlp-rwanda/tech-strikers-bn-backend.git`
- run `cd tech-strikers-bn-backend `
- run `git checkout ft-multi-languages-175636777 `
- run `npm run dev`
- Use postman on `localhost:3000/home` for testing
- In headers add `Accept-Language` with key `en` or `fr` or `ki` depending on preferred language
#### Any background context you want to provide?
- "en": English
- "fr": French
- "ki": Kinyarwanda
#### What are the relevant pivotal tracker stories?
[#175636777](https://www.pivotaltracker.com/story/show/175636777)
#### Screenshots (if appropriate)
image
image
image

### CODE REVIEW

**Review by tytyne — CHANGES_REQUESTED** [review]
can you please add new line to languages/fr.json  and ki.json files!?

**Review by Thiernope — CHANGES_REQUESTED** [review]
Ensure there is one line at the end of languages/fr.json and languages/ki.json files

**Review by tytyne — COMMENTED** [review]
Plz add the proper description of this PR .
- for instance git checkout `name of the branch `
- the way we  can see other languages(French and Kinyarwanda)

**Comment by fabriceumuhire** [comment]
> can you please add new line to languages/fr.json and ki.json files!?

Done

**Comment by fabriceumuhire** [comment]
> Ensure there is one line at the end of languages/fr.json and languages/ki.json files

Done!

**Comment by kipropbrian** [comment]
Kindly rebase and translate all messages currently in the app

File: src/routes/api/users.js [hunk 1]
```diff
@@ -4,25 +4,25 @@ const router = express.Router();
 
 router.get("/user", (req, res, next) => {
   res.status(200).json({
-    message: "successfully sent query"
+    message: res.__("successfully sent query")
   });
```
Comment (Duncanwar) [inline src/routes/api/users.js]: Why are you adding 2 underscore ?

**Review by Duncanwar — COMMENTED** [review]
It looks well for me. After explaining the reason for those underscore. I will approve it

File: src/routes/api/users.js (hunk 1 above)
Comment (bigelio) [reply src/routes/api/users.js]: Can you please do rebase first?

Comment (fabriceumuhire) [reply src/routes/api/users.js]: @Duncanwar for the message to be translated.

Comment (fabriceumuhire) [reply src/routes/api/users.js]: @BiGelio Done.

**Pull request merged**